examples/Extensions/RemoveEntireSitelinkCampaignExtensionSetting.php

```diff
@@ -0,0 +1,233 @@
+
+        // Issues a mutate request to remove the campaign extension setting.
+        $extensionFeedItemServiceClient =
+            $googleAdsClient->getCampaignExtensionSettingServiceClient();
+        $response = $extensionFeedItemServiceClient->mutateCampaignExtensionSettings(
+            $customerId,
+            [$extensionFeedItemOperation]
+        );
+
+        // Prints the resource name of the removed campaign extension setting.
+        /** @var CampaignExtensionSetting $removedCampaignExtensionSetting */
+        $removedCampaignExtensionSetting = $response->getResults()[0];
+        printf(
+            "Removed a campaign extension setting with resource name: '%s'.%s",
+            $removedCampaignExtensionSetting->getResourceName(),
+            PHP_EOL
+        );
+
+        // 2) Removes all the extension feed items of the previously removed campaign extension
+        // setting.
+
+        foreach ($extensionFeedItemResourceNames as $extensionFeedItemResourceName) {
+            // Creates an operation to remove the extension feed item.
+            $extensionFeedItemOperation = new ExtensionFeedItemOperation();
+            $extensionFeedItemOperation->setRemove($extensionFeedItemResourceName);
+
+            // Issues a mutate request to remove the extension feed item.
+            $extensionFeedItemServiceClient =
+                $googleAdsClient->getExtensionFeedItemServiceClient();
+            $response = $extensionFeedItemServiceClient->mutateExtensionFeedItems(
+                $customerId,
+                [$extensionFeedItemOperation]
+            );
+
+            // Prints the resource name of the removed extension feed item.
+            /** @var ExtensionFeedItem $removedExtensionFeedItem */
+            $removedExtensionFeedItem = $response->getResults()[0];
+            printf(
+                "Removed an extension feed item with resource name: '%s'.%s",
+                $removedExtensionFeedItem->getResourceName(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Returns all sitelink extension feed items associated the specified campaign extension
+     * setting.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the client customer ID
+     * @param int $campaignId the campaign ID to get its all sitelink extension feed items
+     * @return string[] the array of resource names of extension feed items
+     */
+    private static function getAllSitelinkExtensionFeedItems(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $campaignId
+    ): array {
+        $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+        // Creates a query that retrieves all campaigns.
+        $query = sprintf(
+            "SELECT campaign_extension_setting.campaign,"
+            . " campaign_extension_setting.extension_type,"
```

Comment:
nit: I usually find it more readable when the space is at the end of each line and not at the beginning.

Reply:
I'm fine with both but changed anyway to match other examples.